pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -93,6 +113,21 @@ func (c *Cluster) Start() error {
 		logger.Warningf("useAllNodes is set to false and no nodes are specified, no OSD pods are going to be created")
 	}
 
+	// ensure the orchestration status map is created
+	if err := c.makeOrchestrationStatusMap(); err != nil {
+		return fmt.Errorf("failed to make OSD orchestration status config map: %+v", err)
+	}
+
+	// disable scrubbing during orchestration and ensure it gets enabled again afterwards
+	if o, err := client.DisableScrubbing(c.context, c.Namespace); err != nil {
+		logger.Warningf("failed to disable scrubbing: %+v. %s", err, o)
+	}
+	defer func() {
+		if o, err := client.EnableScrubbing(c.context, c.Namespace); err != nil {
```

Comment:
before scrubbing is enabled we don't need to check that all the osds are healthy?